production/helm/loki-stack/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: "v1"
 name: loki-stack
-version: 0.30.0
+version: 0.31.0
```

Comment:
```suggestion
version: 0.31.1
```

Reply:
You have a point there, will update accordingly.